### DISCUSSION TITLE
Fixer that converts to `get_debug_type()` under PHP8

### DISCUSSION THREAD

**Discussion by keradus** [PHP-CS-Fixer/PHP-CS-Fixer#7851] [Feature ideas]
ref https://github.com/symfony/symfony/pull/35945

requested by @nicolas-grekas

**Comment by derrabus** [comment]
To brainstorm this, what kind of expression would that fixer convert to `get_debug_type()`? This tenary expression probably:

```php
is_object($var) ? get_class($var) : gettype($var)
```

Anything else?

**Comment by kubawerlos** [comment]
Separate `get_class($var)` or `gettype($var)` too.
